Backend/app/common/PropertiesManager.py

```diff
@@ -91,7 +93,7 @@ def _load_architecture(self):
         if not architecture_type:
             architecture_type = DEFAULT_ARCHITECTURE
             self.__setattr__(ARCHITECTURE_ENV_NAME, DEFAULT_ARCHITECTURE)
```

Comment:
I think this lane has no effect, because line `self.__setattr__(ARCHITECTURE_ENV_NAME, architecture_type)` is already loading the architecture name.

Reply:
you are taking about this line right?
architecture_type = DEFAULT_ARCHITECTURE

Reply:
or are talking about the whole if block ? @AntonioMrtz

Reply:
> you are taking about this line right? architecture_type = DEFAULT_ARCHITECTURE

nope, `self.__setattr__(ARCHITECTURE_ENV_NAME, DEFAULT_ARCHITECTURE)`.